prerequisites.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
+
+echo ""
+echo "Welcome! Before you can run our tool, you need to make sure you have all the packages and libraries installed !"
+
+echo ""
+#checks if caliper is installed under /bin, if not it will be installed and also runs the caliper bind to conigure ethereum globally
+
+cd cp_ws_1920/bin
+
+if hash "caliper" 2>/dev/null; then
```

Comment:
I think you should check first if npm is installed before checking caliper because you need npm to install caliper.

Reply:
right! thx